docker-quay/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Copyright 2022-2023 Tigris Data, Inc.
```

Comment:
This file is no longer necessary, correct?

Reply:
That is right. It was only needed for Quay's `build-push-action@v1` action